Source/WebCore/platform/graphics/texmap/TextureMapper.h

```diff
@@ -70,6 +71,7 @@ class TextureMapper {
     void drawNumber(int number, const Color&, const FloatPoint&, const TransformationMatrix&);
 
     WEBCORE_EXPORT void drawTexture(const BitmapTexture&, const FloatRect& target, const TransformationMatrix& modelViewMatrix = TransformationMatrix(), float opacity = 1.0f, AllEdgesExposed = AllEdgesExposed::Yes);
+    void drawTextureFragment(const BitmapTexture& sourceTexture, const FloatRect& sourceRect, const FloatRect& targetRect);
```

Comment:
Since this is only used by damage I would add `#if ENABLE(DAMAGE_TRACKING)` to make it explicit.